test/grains_cases_test.rb

```diff
@@ -0,0 +1,87 @@
+require_relative 'test_helper'
+
+class GrainsTest < Minitest::Test
```

Comment:
I really like that you're unit testing the `Case` classes. :+1: 

Hopefully doing this will help you realise which methods should be part of some kind of `BaseCase` class that could be inherited from rather than always starting from `OpenStruct`